app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -56,13 +57,26 @@ import java.util.logging.LogRecord
 class InstallerViewModel(
     private val input: Destination.Installer
 ) : ViewModel(), KoinComponent {
+    private val logger = ManagerLogger()
     private val app: Application by inject()
     private val fs: Filesystem by inject()
     private val pm: PM by inject()
     private val workerRepository: WorkerRepository by inject()
     private val installedAppRepository: InstalledAppRepository by inject()
     private val rootInstaller: RootInstaller by inject()
 
+    val installerStatusDialogModel = object : InstallerStatusDialogModel {
+        override var packageInstallerResult by mutableStateOf<PackageInstallerResult?>(null)
+
+        override fun reinstall() {
+            this@InstallerViewModel.reinstall()
+        }
+
+        override fun install() {
+            install(InstallType.DEFAULT) // Always default for now.
+        }
```

Comment:
This is better as long as the formatter permits it:
```suggestion
        override fun reinstall() = this@InstallerViewModel.reinstall()

        override fun install() = install(InstallType.DEFAULT) // Always default for now.
```